inscriptions/ISic000527.xml

```diff
@@ -190,14 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/>Diis M<supplied reason="lost">anibus</supplied>
+                    <lb n="2"/><name><expan><abbr>P</abbr><ex>ublio</ex></expan></name> <g ref="#interpunct">·</g> <name>Stationam</name> <gap reason="lost" extent="unknown" unit="character"/> <name><supplied reason="lost">Sta</supplied>
+                    <lb n="3" break="no"/>tius</name> <name><supplied reason="lost">Qu</supplied>artus</name> <g ref="#interpunct">·</g> et <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <supplied reason="lost">sibi</supplied>
+                    <lb n="4"/>et <g ref="#interpunct">·</g> <name>Salviae</name> <g ref="#interpunct">·</g> <name><expan><abbr>C</abbr><ex>ai</ex></expan></name> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>iliae</ex></expan> <g ref="#interpunct">·</g> <name>Felicul<supplied reason="lost">ae</supplied></name> <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <supplied reason="lost">uxori</supplied>
+                    <lb n="5"/>posterisque suis <g ref="#interpunct">·</g> <expan><abbr>l</abbr><ex>ocus</ex></expan> <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <supplied reason="lost"><expan><abbr>d</abbr><ex>atus</ex></expan></supplied> <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <supplied reason="lost"><expan><abbr>d</abbr><ex>ecreto</ex></expan></supplied> <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <supplied reason="lost"><expan><abbr>d</abbr><ex>ecurionum</ex></expan></supplied>
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>Text: CIL</note></app></listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#AA">
+                <p>To the shades of the underworld. For Publius Stationam[...?] [...] Statius Quartus, and his children, and for Salvia Felicula, daughter of Caius, his wife, and for those who come afterwards. The place for it was given by the decree of the decurions</p>
```

Comment:
So, Publius.... set it up both for himself and for Salvia Felicula his wife and for his children.